backend/src/test/java/ai/verta/modeldb/MergeTest.java

```diff
@@ -395,6 +395,28 @@ static Blob getDatasetBlobFromPath(String path, long size) {
         .build();
   }
 
+  static Blob getDatasetBlobFromPathMultiple(String path, long size) {
```

Comment:
I don't know the purpose of this. otherwise LGTM.

Reply:
util  function to generate the bug report case